tensorflow/tools/tf_sig_build_dockerfiles/devel.usertools/gpu.bazelrc

```diff
@@ -15,6 +15,21 @@ build --action_env=CACHEBUSTER=565341047
 # Build options for GPU Linux
 build --config=release_gpu_linux
 
+# ROCM: Set up compilation ROCM version and paths
+build:rocm --linkopt="-fuse-ld=gold"
```

Comment:
I think we want to use ldd for the linker

Reply:
I think I had some issues getting lld on all docker images. I will have to check that.